python/Ice/context/client/main.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# Copyright (c) ZeroC, Inc.
+
+import Ice
+import asyncio
+import getpass
+import sys
+
+# Slice module VisitorCenter in Greeter.ice maps to Python module VisitorCenter.
+from VisitorCenter import GreeterPrx
+
+async def main():
+    # Set the Ice.ImplicitContext property to "Shared" before calling Ice.initialize.
+    # This is only necessary for the implicit context API (see below).
+    initData = Ice.InitializationData()
+    initData.properties = Ice.createProperties(sys.argv)
+    initData.properties.setProperty("Ice.ImplicitContext", "Shared")
+
+    # Configure the Ice runtime to use asyncio.
```

Comment:
While this is consistent with the comment on initialize below, I am concerned this is confusing.

The Ice runtime and an Ice communicator are not synonymous. For example, a "process logger" is definitely about the Ice runtime, while this asyncio enabling is only for a single commmunicator, not the Ice runtime as a whole.

For another PR!